packages/appkit/src/client.ts

```diff
@@ -1106,6 +1106,11 @@ export class AppKit {
         }
 
         return '0x'
+      },
+      walletGetAssets: async (params: object | readonly unknown[]) => {
+        const adapter = this.getAdapter(ChainController.state.activeChain as ChainNamespace)
```

Comment:
We shouldn't cast the `activeChain` to `ChainNamespace` as it's bad practice (we are doing this also in many places, I'll add ticket for tech debt). The `ChainController.state.activeChain` can be undefined and this undefined should be handled in `getAdapter`